src/gfx/shaders/VolumeMaterial.js

```diff
@@ -2,10 +2,10 @@
 /* eslint-disable guard-for-in */
 import * as THREE from 'three';
 import volumeFrag from './Volume_frag.glsl';
+import globalSettings from '../../settings';
```

Comment:
No way 😄 We must use `settings` here and change the local variable name instead.

Reply:
Know that now